src/App.css

```diff
@@ -1,4 +1,283 @@
-.author-wrapper {
-  background-color: white;
-  margin-left: 2%;
+:root {
+  --border-color: #f1f1f1;
+  --backround-color: #fafafa;
+  --primary-color: #0290ff;
+  --accent-color: #8a8a8a;
+  --max-width: 1000px;
+}
+
+.nav-container {
+  display: grid;
+  place-items: center;
+  border-bottom: 2px solid var(--border-color);
+  margin-bottom: 2%;
+  margin-left: -5rem;
+}
```

Comment:
- [ ] This is a style ,milestone so we have to make sure the styling is close to the given design, currently your nav is not aligned to the body of the booklist, kindly make it so
your design :point_down: 
![image](https://github.com/rbd3/bookstore/assets/66097371/1a3b21a4-aaa5-4e55-8806-ae78646b022b)
given design :point_down: 
![image](https://github.com/rbd3/bookstore/assets/66097371/5652e9f8-d24d-4510-b95e-2b0c51e68ba8)